iree/turbine/kernel/wave/wave.py

```diff
@@ -694,7 +694,7 @@ def test_execute(self, args, kwargs):
                 kernel_hash=kernel_hash,
             )
 
-        return mb
+        return mb.module_op.get_asm()
```

Comment:
Going to defer on @harsh-nod  on this one whether or not we ever need to return the mb, if not this makes sense

Reply:
Lets go with returning asm for now, we can revisit later.